lib/src/components/mini_chevron_left.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+/// chevron Icon
+class MiniChevronLeft extends StatelessWidget {

```

Comment:
Can't we reuse `ChevronDown`?

Reply:
We could either work around this by just making `width` and `height` smaller, or we could add a `scale` argument to `Chevron` and reuse it.
Otherwise it's duplicate functionality.
<img width="998" alt="image" src="https://user-images.githubusercontent.com/40931732/187712984-c58a4447-cfe9-4339-91e8-60e44b67638c.png">

Reply:
i moved arrow to center of circle